### PR TITLE
add NFT detail

### PR DESCRIPTION
**Pull request by hydraCy**

### CODE REVIEW

File: Maskbook/Scene/Wallet/NFTDetail/NFTActionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            }
+        }
+    
+    private func saveImage(image: UIImage) {
+            PHPhotoLibrary.shared().performChanges({
+                PHAssetChangeRequest.creationRequestForAsset(from: image)
+            }, completionHandler: {(isSuccess, error) in
+                DispatchQueue.main.async {
+                    if isSuccess {
+                       // to do  toast
```
Comment (sxiaojian88) [inline Maskbook/Scene/Wallet/NFTDetail/NFTActionViewController.swift:190]: You can do the toast with Toast.swift

File: Maskbook/Scene/Wallet/NFTDetail/View/NFTActionTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+//  Maskbook
+//
+//  Created by caiyu on 2022/2/23.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import UIKit
+
+class NFTActionTableViewCell: UITableViewCell {
+    static let reuseIdentifier = "NFTActionTableViewCell"
```
Comment (sxiaojian88) [inline Maskbook/Scene/Wallet/NFTDetail/View/NFTActionTableViewCell.swift:12]:
```suggestion
   
```

File: Maskbook/Error/MaskError.swift [hunk 3]
```diff
@@ -57,7 +59,8 @@ extension WalletSendError: LocalizedError {
             return "Invalid address"
             
         case .ethereumError(let error):
-            return error.localizedDescription
+            let web3error = error as! Web3Error
```
Comment (foxsin10) [inline Maskbook/Error/MaskError.swift:62]: Avoid force unwrap here, use  `(error as? Web3Error)?.errorDescription  ?? ""` instead.

File: Maskbook/Scene/Wallet/Balance/View/AccountCardView.swift [hunk 4]
```diff
@@ -43,11 +44,11 @@ class AccountCardView: UIView {
     private var stackViewGradientLayer: CAGradientLayer = {
         let layer1 = CAGradientLayer()
         layer1.colors = [
-            UIColor.white.withAlphaComponent(0.08).cgColor,
```
Comment (foxsin10) [inline Maskbook/Scene/Wallet/Balance/View/AccountCardView.swift:46]: Conflicts with [this pull request](https://github.com/DimensionDev/Mask-iOS/pull/24)

**Review by foxsin10 — CHANGES_REQUESTED** [review]

**Review by sxiaojian88 — APPROVED** [review]

**Comment by jk234ert** [comment]
@hydraCy plz fix the conflicts

**Pull request merged**